src/Adapter/Hook/HookDispatcher.php

```diff
@@ -44,7 +44,7 @@
  * If the event is a RenderingHookEvent, then the final result is
  * an array of contents accessed from $event->getContent().
  */
-class HookDispatcher extends EventDispatcher implements HookDispatcherInterface
```

Comment:
this dependency needs to be removed (maybe just copy/paste missing functions from event dispatcher ?